client/src/components/CourseNavHome/CourseNavHome.js

```diff
@@ -0,0 +1,224 @@
+      backgroundColor: colors.red.dark,
+    },
+    top: {
+        backgroundColor: colors.red.dark,
+        height: 160,
+        display: 'flex',
+        borderBottomLeftRadius: 20,
+        borderBottomRightRadius: 20
+    },
+    manageBar: {
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'space-between'
+    },
+    selected: {
+
+    }
+  })
+  const ManageView =({
+    languageName,
+    languageDescription,
+    unitsNumber,
+    buttonText,
+    rightIconName,
+    unitsName,
+    lessonNumber,
+    statusTag,
+
+  })
+  const data = [
+    {
+      _id: 'abcdef',
+      title: 'Unit 1',
+      lessons: '2 Lessons',
+      indicatorType: INDICATOR_TYPES.COMPLETE,
+    },
+    {
+      _id: 'aenasdas',
+      title: 'Unit 2',
+      lessons: '7 Lessons',
+      indicatorType: INDICATOR_TYPES.INCOMPLETE,
+    },
+    {
+      _id: 'asdnemsa',
+      title: 'Unit 3',
+      lessons: '4 Lessons',
+      indicatorType: INDICATOR_TYPES.COMPLETE,
+    },
+    {
+      _id: 'mehjasjd',
+      title: 'Unit 5',
+      lessons: '3 Lessons',
+      indicatorType: INDICATOR_TYPES.COMPLETE,
+    },
+    {
+      _id: 'asdnemsa',
+      title: 'Unit 3',
+      lessons: '4 Lessons',
+      indicatorType: INDICATOR_TYPES.COMPLETE,
+    },
+    {
+      _id: 'mehjasjd',
+      title: 'Unit 5',
+      lessons: '3 Lessons',
+      indicatorType: INDICATOR_TYPES.COMPLETE,
+    },
+    {
+      _id: 'asdnemsa',
+      title: 'Unit 3',
+      lessons: '4 Lessons',
+      indicatorType: INDICATOR_TYPES.COMPLETE,
+    },
+    {
+      _id: 'mehjasjd',
+      title: 'Unit 5',
+      lessons: '3 Lessons',
+      indicatorType: INDICATOR_TYPES.COMPLETE,
+    },
+    {
+      _id: 'asdnemsa',
+      title: 'Unit 3',
+      lessons: '4 Lessons',
+      indicatorType: INDICATOR_TYPES.COMPLETE,
+    },
+    {
+      _id: 'mehjasjd',
+      title: 'Unit 5',
+      lessons: '3 Lessons',
+      indicatorType: INDICATOR_TYPES.COMPLETE,
+    },
+    
+  ]
+
+  const CourseNavHome = ({navigation}) => {
+    return (
+        <>
+        <View
+        style={styles.top}
+        >
+            <Text
+           style={{
+            fontFamily: 'GT_Haptik_bold',
+            
+          }}
+            color="white.dark"
+            fontSize={35}
+            paddingLeft={5}
+            paddingTop={5}
+    >
+      {languageName}
+      </Text>
+      <Text
+      style={{
+        fontFamily: 'GT_Haptik_bold',
+        
+      }}
+            color="white.dark:alpha.40"
+            fontSize={'xl'}
+            lineHeight={20}
+            paddingLeft={5}
+            paddingRight={5}
+            adjustsFontSizeToFit={true}
+    >
+      {languageDescription}
+     </Text>
+
+        </View>
+<View
+style={styles.manageBar}
+>
+    <Text
+    style={{
+        fontFamily: 'GT_Haptik_bold',
+        
+      }}
+      fontSize={23}
+      paddingTop={3}
+      paddingLeft={5}
+      >    
+{unitsNumber} 4 Units
```

Comment:
number of "units" or "lessons"... is determined from length of data, name of `Units `is a prop